packages/components/ContextualMenu/src/ContextualMenu.tsx

```diff
@@ -78,16 +80,12 @@ export const ContextualMenu = compose<ContextualMenuType>({
         setInitialFocus: shouldFocusOnMount,
         ...rest,
       },
-      container: Platform.select({
```

Comment:
retro nit + unsure, is ```...rest``` above necessary if we aren't manipulating the props in this fn?

Reply:
It's a convention to pass unrelated props down when creating a Higher Order Component: https://reactjs.org/docs/higher-order-components.html#convention-pass-unrelated-props-through-to-the-wrapped-component

We've had bugs before where say, tooltip wasn't set on CheckBox, because we didn't explicitly forward it to the root slot. Adding `...rest` in general solves that issue.